src/chrome/content/rules/beeline.kz.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+beeline.kz/: Redirect for 'https://beeline.kz/' missing Location
+app.beeline.kz non-2xx http code
+nomer.beeline.kz mismatch
+podarok.beeline.kz self signed
+like.beeline.kz timed out
+video.beeline.kz timed out
+love.beeline.kz timed out
+music.beeline.kz mismatch
+fitness.beeline.kz timed out
+kids.beeline.kz mismatch
+football.beeline.kz timed out
+wap.beeline.kz timed out
+book.beeline.kz timed out
+m.beeline.kz timed out
+777.beeline.kz self signed
+-->
+<ruleset name="beeline.kz">
+	<target host="beeline.kz" />
+	<target host="www.beeline.kz" />
+	<target host="4g.beeline.kz" />
+	<target host="money.beeline.kz" />
+	<target host="my.beeline.kz" />
+	<target host="oplata.beeline.kz" />
+	<target host="vip.beeline.kz" />
+	<target host="uslugi.beeline.kz" />
+	<target host="help.beeline.kz" />
+	<target host="my.internet.beeline.kz" />
```

Comment:
Nope, since sorting is

> from the top-level domain at the right
    reading left, then moving '^' and 'www' to the top of their group.

https://github.com/aboul3la/Sublist3r/commit/80a6458077d740c48c21e4b6a936aa67772664d8#diff-68d6e240bb72a5057a7a635f1bcdeda8L101
https://github.com/aboul3la/Sublist3r/commit/09a2b9cdaf020a7dac6701765a846807ef6db814#diff-68d6e240bb72a5057a7a635f1bcdeda8R82